upload.sh

```diff
@@ -1,44 +1,101 @@
 #!/bin/sh
 
-# Get tag from environment
+# Get release tag from environment and checkout branch.
 tag=$(basename $GITHUB_REF)
+git checkout tags/$tag
 
-# Get action that triggered release event
+# Get package version.
+version=$(python setup.py --version)
+
+# Check if release tag matches the package version.
+pip install --quiet packaging>=17.0
```

Comment:
`packaging>=17.0` needs to be enclosed in quotes so the `>` is not interpreted as a command to redirect the output

Reply:
I see. Updated to enclose in quotes.